src/pybind/mgr/orchestrator/module.py

```diff
@@ -1188,6 +1188,46 @@ def _apply_rgw(self,
                 out = to_format(data, format, many=True, cls=None)
         return HandleCommandResult(stdout=out)
 
+    @_cli_write_command(
+        'orch apply haproxy',
+        'name=placement,type=CephString,req=false '
+        'name=dry_run,type=CephBool,req=false '
+        'name=format,type=CephChoices,strings=plain|json|json-pretty|yaml,req=false '
```

Comment:
We do not have any big output from this command. Is just to create things ... so i think you can remove the format parameter